src/main/java/org/kuali/kra/proposaldevelopment/web/struts/action/ProposalDevelopmentActionsAction.java

```diff
@@ -332,19 +334,27 @@ else if ( variableKey.startsWith( KewApiConstants.RECEIVE_FUTURE_REQUESTS_BRANCH
 	private boolean canGenerateMultipleApprovalRequests( RoutingReportCriteria reportCriteria, String loggedInPrincipalId, String currentRouteNodeNames ) throws Exception {
 		WorkflowDocumentActionsService info = GlobalResourceLoader.getService( "rice.kew.workflowDocumentActionsService" );
 		DocumentDetail results1 = info.executeSimulation( reportCriteria );
-		int crtUserActiveActionRequestsCount = 0;
+		int hierarchyActionRequests = 0;
 		for ( ActionRequest actionRequest : results1.getActionRequests() ) {
 			if ( actionRequest.isPending()
 			        && actionRequest.getActionRequested().getCode().equalsIgnoreCase( KewApiConstants.ACTION_REQUEST_APPROVE_REQ ) 
 			        && recipientMatchesUser( actionRequest, loggedInPrincipalId ) )	{
-				crtUserActiveActionRequestsCount++;
-				LOG.debug("MultipleApprovalRequests: actionRequestId="+actionRequest.getId()+" actionRequestAnnotation="+actionRequest.getAnnotation()+" crtUserActiveActionRequests="+crtUserActiveActionRequestsCount);
-				if ( crtUserActiveActionRequestsCount >1 ){
-					return true;
-				}
+					if ( StringUtils.contains( currentRouteNodeNames, actionRequest.getNodeName() ) ){
+
+						if ( NODE_NAME_HIERARCHY_REQUEST.equalsIgnoreCase(actionRequest.getNodeName())){
+							hierarchyActionRequests++;
+							LOG.debug("MultipleApprovalRequests: actionRequestId="+actionRequest.getId()+" actionRequestAnnotation="+actionRequest.getAnnotation()+" hierarchyActionRequests="+hierarchyActionRequests);
+							if ( hierarchyActionRequests > 1)
+								return true;
+						}
+
+					} else {
+						LOG.debug("MultipleApprovalRequests: actionRequestId="+actionRequest.getId()+" actionRequestAnnotation="+actionRequest.getAnnotation()+" crtUserActiveActionRequests=1");
```

Comment:
Debug refs crtUserActiveActionRequests, but is now hierarchyActionRequests.

Reply:
Oh indeed. I can take out that debug for good.